Makefile

```diff
@@ -1,3 +1,7 @@
+# Used in CI to avoid running with yarn
+NODE_TOOL_PREFIX?=yarn
+export NODE_TOOL_PREFIX
```

Comment:
I don't love this. A big part of my job is making sure we keep things understandable for ourselves and others in the future, and a big part of the is consistency and predictability. As someone looking at this in the future, I'd have to ask myself "why doesn't _every_ instance of `yarn` in this file work the same way"?

Would a better alternative be to execute everything as `node node_modules/.bin/...`?

Reply:
`workspaces focus` works great -- good idea. I never was aware of that feature.

Reply:
Changed in [23bd1ed](https://github.com/PrairieLearn/PrairieLearn/pull/11909/commits/23bd1ed5e0f79044cad2daf0f2a3ecaf2277bb9b).